pkg/build/strategies/layered/layered.go

```diff
@@ -48,7 +54,7 @@ func New(config *api.Config, scripts build.ScriptsHandler, overrides build.Overr
 	}, nil
 }
 
-//getDestination returns the destination directory from the config
+// getDestination returns the destination directory from the config.
 func getDestination(config *api.Config) string {
```

Comment:
I'm missing equivalent of `COALESCE` (from SQL) here :-( Or something like `strings.FirstNonEmpty()` method because in many places we're using code like this:
```go
result := cfg.Property
if len(result) > 0 {
    result = "default value"
}
```
I would like to write it like this:
```
result := strings.firstNonEmpty(cfg.Property, "default value")
```

(It's just an idea.)